test/controllers/organizations/staff/adoption_application_reviews_controller_test.rb

```diff
@@ -71,6 +71,17 @@ class Organizations::Staff::AdoptionApplicationReviewsControllerTest < ActionDis
         end
       end
     end
+
+    context "#show" do
```

Comment:
We are not actually authorizing requests to this route yet in the controller. I think this is passing because it's just checking if the user authorization on that policy for `manage?`, which is should pass. See my note about authorization in the controller. We just need to call `authorize!...` and add `show` to the `set_alias` method in the policy class.

Reply:
Have updated show method with authorize code and added alias in policy class